yt_dlp/extractor/ettutv.py

```diff
@@ -41,7 +41,7 @@ def _real_extract(self, url):
                 'device': 'desktop',
             })
 
-        stream_response = self._download_json(player_settings['streamAccess'], video_id, data={})
```

Comment:
I think this should still work? though setting it to empty byte makes it more clear

Reply:
it worked pre-network rework. it throws now